omnibus/files/private-chef-cookbooks/private-chef/recipes/bootstrap.rb

```diff
@@ -1,70 +1,59 @@
 #
-# Author:: Adam Jacob (<adam@opscode.com>)
-# Copyright:: Copyright (c) 2011 Opscode, Inc.
+# Author:: Adam Jacob <adam@chef.io>
+# Copyright:: Copyright (c) 2011-2015 Chef Software, Inc.
 #
 # All Rights Reserved
 #
 
-require 'securerandom'
-
 # Enterprise Addon Install
-# On a new install, download and install all of the Chef
-# addons from package repositories. Only perform this
-# action during a fresh install, and not during an upgrade.
+include_recipe "private-chef::opscode-erchef"
```

Comment:
Because this fails if we don't: 

https://github.com/chef/chef-server/pull/677/files#diff-1f3de27173d5e39c86242170696a4f19R48

Maybe I could just add the service declaration instead? Will that cause trouble if I do?